pt-BR/mail/chrome/messenger/folderpane.dtd

```diff
@@ -34,10 +34,10 @@
  the terms of any one of the MPL, the GPL or the LGPL.
 
  ***** END LICENSE BLOCK ***** -->
-<!ENTITY nameColumn.label   "Nome"> 
-<!ENTITY unreadColumn.label "Não lidas"> 
-<!ENTITY totalColumn.label "Total"> 
-<!ENTITY folderSizeColumn.label "Tamanho"> 
-<!ENTITY accounts.label "Accounts">
+<!ENTITY nameColumn.label   "Nome">
+<!ENTITY unreadColumn.label "Não lidas">
```

Comment:
@brunotiussi I agree. Therefore, you must be consistent and chose either to use "não lida/s" or "não-lida/s".